app/frontend/filters/reviewer_filter.component.js

```diff
@@ -0,0 +1,36 @@
+import { Component }          from 'react';
+import { bindActionCreators } from 'redux';
+import { connect }            from 'react-redux';
+
+import FilterOptionGroup      from './filter_option_group.component';
+import FilterOption           from './filter_option.component';
+
+import { setFilterGroup }     from './filters.actions';
+
+class ReviewerFilter extends Component {
+  render() {
+    if (this.props.session.is_reviewer) {
```

Comment:
I keep thinking that eventually we'll have to somehow expose `cancancan`'s permissions into an API endpoint.